addon/components/frost-button.js

```diff
@@ -222,5 +223,12 @@ export default Ember.Component.extend({
     if (!this.get('disabled') && _.isFunction(this.attrs['onClick'])) {
       this.attrs['onClick'](this.get('id'))
     }
+  }),
+
+  _onFocus: Ember.on('focusIn', function (e) {
+    // If an onFocus handler is defined, call it
+    if (this.attrs.onFocus) {
+      this.attrs.onFocus()
+    }
```

Comment:
@sandersky Good points - I was also wondering if we could alias in on top of the Ember events - e.g. map `onFocusIn` (our event format) to `focus-in` (Ember's event format) - might be something we could mess with globally via a Component re-open so that we could remap all of the events in one shot